### PR TITLE
ci: use net-http gem in tests

### PR DESCRIPTION
**Pull request by GSVarsha**

### CODE REVIEW

**Review by Ferenc- — APPROVED** [review]
LGTM

File: .tekton/run_unittests.sh [hunk 1]
```diff
@@ -50,6 +50,9 @@ echo    "with dependencies in '${BUNDLE_GEMFILE}'"
 gem update --system > /dev/null
 echo "Gem version $(gem --version)"
 
+# Verify if net-http is included in the installation (no separate gem needed)
```
Comment (Ferenc-) [inline .tekton/run_unittests.sh:53]: I find this comment a bit misleading. The gem is always included, no need to verify that.
And it is always a "separate" gem, just like the rest.
The question is whether the included one is the latest or an older one.

File: test/instrumentation/net_http_test.rb [hunk 2]
```diff
@@ -4,6 +4,8 @@
 require 'test_helper'
 require 'support/apps/http_endpoint/boot'
 
+require 'net/http'
```
Comment (Ferenc-) [inline test/instrumentation/net_http_test.rb:7]: Is this BTW strictly necessary?

**Review by Ferenc- — CHANGES_REQUESTED** [review]

File: test/instrumentation/net_http_test.rb (hunk 2 above)
Comment (GSVarsha) [reply test/instrumentation/net_http_test.rb]: Removed it in the latest commit.

File: .tekton/run_unittests.sh (hunk 1 above)
Comment (GSVarsha) [reply .tekton/run_unittests.sh]: Modified the comment to [this](https://github.com/instana/ruby-sensor/pull/388/files#diff-7634e6a9f9a1d293732ac9239731ed4fbbafbf1ad312b79b4d5486f7ce2369f8R53).

> The question is whether the included one is the latest or an older one.
It is the latest version (0.4.1)

**Review by pvital — APPROVED** [review]
It looks good to me.

**Review by Ferenc- — APPROVED** [review]
Approved

**Pull request merged**